dd-sdk-android-core/src/main/kotlin/com/datadog/android/ndk/internal/DatadogNdkCrashHandler.kt

```diff
@@ -44,9 +43,6 @@ internal class DatadogNdkCrashHandler(
     internal var lastNetworkInfo: NetworkInfo? = null
```

Comment:
Probably we can remove `lastNetworkInfo` and `lastUserInfo` since they were used only for Logs? We can also remove the part where we write them to the disk.